### PR TITLE
feat: Ingest & Classification Functions

### PR DESCRIPTION
**Pull request by HarryET**
- Adds signature verification to ingest

### CODE REVIEW

File: supabase/functions/classify_feedback/index.ts [hunk 1]
```diff
@@ -1,12 +1,123 @@
...
+
+const supabase = createClient(
+  Deno.env.get('SUPABASE_URL') ?? '',
+  Deno.env.get('SUPABASE_ANON_KEY') ?? ''
+)
+
+const supabaseSecret = createClient(
+  Deno.env.get('SUPABASE_URL') ?? '',
+  Deno.env.get('SUPABASE_SERVICE_ROLE_KEY') ?? ''
+)
```
Comment (Olyno) [inline supabase/functions/classify_feedback/index.ts:18]: Maybe should it be better to let the value by default, and let supabase throw the error if necessary? (same for others)

Comment (HarryET) [reply supabase/functions/classify_feedback/index.ts:18]: It shows IDE errors, and that's what all the docs/examples do and recommend. But yes typically I would agree with you.

Comment (Olyno) [reply supabase/functions/classify_feedback/index.ts:18]: Alright, if the docs/examples show it, just let's do it

**Review by Olyno — COMMENTED** [review]
Looks good to me

**Review by Olyno — APPROVED** [review]

**Comment by HarryET** [comment]
This will be closed, all progress will be migrated to @VictorPeralta's feat/twitter-ingest branch

**Pull request closed without merging**